src/main/java/Task.java

```diff
@@ -0,0 +1,19 @@
+// Add the ability to store whatever text entered by the user and display them back to the user when requested.
+public class Task {
+    
+
+    String[] taskList = new String[100];
+    int taskCount = 0;
+
+    public void addTask(String description){
+        taskList[taskCount] = description;
+        taskCount++;
+        System.out.println("\tadded: " + description);
+    }
+
+    public void listTask(){
```

Comment:
Space needed before `{`, and method name should be plural


```suggestion
    public void listTasks() {
```